package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "overlay-plugin",
+  "version": "0.0.0",
+  "type": "module",
+  "types": "./types/index.d.ts",
+  "main": "./lib/index.js",
+  "files": [
+    "dist/",
+    "lib/",
```

Comment:
Doesn't this end up in dist/ after building?

Reply:
(I should also mention that I don't know what goes into publishing an npm package, so please correct me if I've suggested anything weird.  Just pointing out questions I had that might be my own misunderstanding.)

Reply:
> Doesn't this end up in dist/ after building?

yes